src/components/manage/MySongs.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
Please do not import React ...
1. you are not using it
2. Babel does this for us by default